Source/Core/Core/DSP/DSPTables.cpp

```diff
@@ -494,15 +496,15 @@ const std::array<pdlabel_t, 36> regnames =
 
 const char* pdname(u16 val)
 {
-  static char tmpstr[12];  // nasty
+  static char tmpstr[12]{};  // nasty
 
   for (const pdlabel_t& pdlabel : pdlabels)
   {
     if (pdlabel.addr == val)
       return pdlabel.name;
   }
 
-  sprintf(tmpstr, "0x%04x", val);
+  fmt::format_to_n(tmpstr, sizeof(tmpstr) - 1, "0x{:04x}", val);
   return tmpstr;
```

Comment:
Could we make this function return an std::string instead? Returning a pointer to a local static buffer is pretty gross.

Reply:
Done